### PR TITLE
Prevent rendering of the loading image laying off the map (above)

### PR DESCRIPTION
**Pull request by PrimosK**
This pull request fixes the issue #45.

### CODE REVIEW

File: jxmapviewer2/src/main/java/org/jxmapviewer/JXMapViewer.java [hunk 1]
```diff
@@ -298,9 +298,9 @@ private void drawOverlays(final int zoom, final Graphics g, final Rectangle view
 	}
 
 	@SuppressWarnings("unused")
-	private boolean isTileOnMap(int x, int y, Dimension mapSize)
+	private boolean isTileOffTheMap(int x, int y, Dimension mapSize)
 	{
-		return !isNegativeYAllowed && y < 0 || y >= mapSize.getHeight();
```
Comment (msteiger) [inline jxmapviewer2/src/main/java/org/jxmapviewer/JXMapViewer.java]: Why did you remove the check for `isNegativeYAllowed`?

**Comment by PrimosK** [comment]
This fixed the issue for me (as in this case a background is rendered instead of loading image) - if I overlooked something and this isn't a proper fix please simply discard it.

File: jxmapviewer2/src/main/java/org/jxmapviewer/JXMapViewer.java (hunk 1 above)
Comment (PrimosK) [inline jxmapviewer2/src/main/java/org/jxmapviewer/JXMapViewer.java]: This fixed the issue #45 for me (as in this case a background is properly rendered instead of loading image which is later not properly hidden) - if I overlooked something and this isn't a proper fix please simply discard it.
